avocado/core/runners/avocado_instrumented.py

```diff
@@ -1,3 +1,4 @@
+import logging
```

Comment:
Could you improve the git log message description, please?

Reply:
yes, no problem